src/App.scss

```
@@ -1 +1,47 @@
-// styles go here
+.btn {
```

Думаю, така назва класу була би більш понятною)
```suggestion
.button {
```